### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by npnote8**

### CODE REVIEW

File: 02-express-tutorial/routes/people.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const {
+  getPeople,
+  addPerson,
+  getPeopleId,
+  updatePerson,
+  deletePerson,
+} = require("../controllers/people");
+
+router.get("/", getPeople);
+router.get("/:peopleID", getPeopleId);
```
Comment (LukeDias42) [inline 02-express-tutorial/routes/people.js]: You can call the param "peopleID", ID here too for consistency. Or you can change the name of the params on the other HTTP methods. When confronted with inconsistency, you generally try to understand what makes it different, even if there are no differences, which makes future changes a little less smooth.

File: 02-express-tutorial/controllers/products.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const { products } = require("../data");
+
+const getProduct = (req, res) => {
+  return res.json(products);
+};
+
+const getProductId = (req, res) => {
```
Comment (LukeDias42) [inline 02-express-tutorial/controllers/products.js]: You can call this getProductById.

File: 02-express-tutorial/controllers/products.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  const { productID } = req.params;
```
Comment (LukeDias42) [inline 02-express-tutorial/controllers/products.js]: don't forget to change productID to id here too.

File: 02-express-tutorial/controllers/people.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+let { people } = require("../data");
+
+const getPeople = (req, res) => {
+  return res.json({ success: true, data: people });
+};
+
+const getPeopleId = (req, res) => {
```
Comment (LukeDias42) [inline 02-express-tutorial/controllers/people.js]: You could call this getPersonById, to make it more clearer that only one person is returned.

File: 02-express-tutorial/controllers/products.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+const { products } = require("../data");
+
+const getProduct = (req, res) => {
```
Comment (LukeDias42) [inline 02-express-tutorial/controllers/products.js]: This return a list of products, so "getProducts"

**Review by LukeDias42 — APPROVED** [review]

**Comment by LukeDias42** [comment]
I couldn't find any problem, only minor naming issues.

**Comment by npnote8** [comment]
Addressed comments. Thank you

**Pull request merged**